### ISSUE TITLE
Code query

### ISSUE THREAD

**Issue by Tom1123456** [keziyi-buct/TRNN#3]
> Hello, I would like to ask about the scatter plots in your TRN1-1, do they seem to me to be all from the training set
> The code provides an example of plotting a scatterplot which is from the training set. You can also write this code in the test part of the model in experiment. By doing this you can plot the scatterplot of the test set. Just adjust it to your needs, the plotting function is encapsulated in def.  

 _Originally posted by @keziyi-buct in [#2](https://github.com/keziyi-buct/TRNN/issues/2#issuecomment-2745959821)_
Thank you very much for your reply, that is, I saw that the scatterplot of the code you published used the data from the training set, and then I have this problem. Another question is whether the scatterplot in your paper and the results compared with others in Table 3 used data from the test set
